docs/release-notes.txt

```diff
@@ -8,6 +8,53 @@
  Version 0.13
 **************
 
+Version 0.13.4
+==============
+
+**Release Date:** September 16, 2020
+
+**Improvements**
+
+- Warn users when setting ``DistributeConfig`` on an Estimator. Configuring an Estimator via ``tf.distribute.Strategy`` can conflict with how Determined performs distributed training. With this change, Determined will attempt to catch this problem and surface an error message in the experiment logs. An Estimator can still be configured with an empty ``DistributeConfig`` without issue.
+
+- Support configuring default task images, Docker pull policy, and Docker registry credentials via the :ref:`master-configuration` and the :ref:`helm-config`. Note that individual task configurations will continue to overwrite these defaults.
+
+- Add connection checks for dynamic agents. A dynamically provisioned agent will be terminated if it is not actively connected to the master for at least five minutes.
+
+- Remove support for ``dataflow_to_tf_dataset`` in EstimatorTrial. Dataflows should be wrapped using ``wrap_dataset(shard=false)`` instead.
+
+- WebUI: Add middle mouse button click detection on tables to open in a new tab/page.
+
+- WebUI: Improve the trial detail metrics view.
+
+  - Support non-numeric metric values.
+
+  - Default to showing only the searcher metric on initial page load.
+
+  - Add search capability to the metric select filter. This should improve the experience when there is a large set of metrics.
+
+  - Add support for displaying multiple metrics on the metric chart. There is a known issue displaying multiple validation and training metrics that share the same name.
+
+- WebUI: Move TensorBoard sources from a table column into a separate modal.
+
+- WebUI: Reduce the time it takes to load active TensorBoards and notebooks.
+
+**Bug Fixes**
+
+- Improve handling of certain corner cases where distributed training jobs could hang indefinitely.
+
+- Fix an issue where detecting GPU availability in TensorFlow code would cause EstimatorTrial models to OOM.
```

Comment:
EstimatorTrial should be double-backticked.

Reply:
fixed.